tests/functional/spec/radio_optional_with_detail_answer_optional.spec.js

```diff
@@ -0,0 +1,42 @@
+import RadioNonMandatoryPage from "../generated_pages/radio_optional_with_detail_answer_optional/radio-non-mandatory.page";
+import SubmitPage from "../generated_pages/radio_optional_with_detail_answer_optional/submit.page";
+
+describe("Checkbox and Radio item descriptions", () => {
+  beforeEach("load the survey", () => {
+    browser.openQuestionnaire("test_radio_optional_with_detail_answer_optional.json");
+  });
+
+  describe("Given the user is presented with an optional radio answer with optional detail answer", () => {
+    it("When no answer is provided, the expected answer is displayed", () => {
```

Comment:
Just missing the then, and it might help if it said displayed on the summary. Same for the ones below